bgpd/bgp_evpn_vty.c

```diff
@@ -2188,7 +2199,7 @@ static void evpn_show_routes_vni(struct vty *vty, struct bgp *bgp, vni_t vni,
 	}
 
 	/* Walk this VNI's route table and display appropriate routes. */
-	show_vni_routes(bgp, vpn, type, vty, vtep_ip, json);
+	show_vni_routes(bgp, vpn, type, vty, vtep_ip, json, 0);
```

Comment:
This is not clear. Why '0' is passed always ? shouldn't this be based on details flag ?

Reply:
CLI "show bgp l2vpn evpn route vni <> type <>" leads to this call sequence. The expectation from this CLI is to display only summary information hence the value of "detail" flag is passed as 0. Detailed information is displayed only via CLI - "show bgp l2vpn evpn route vni all detail".